firmware/Board.h

```diff
@@ -38,22 +42,32 @@ namespace board
   const int LED = 54;
   const size_t NUM_LEDS = 4;
 
+  const size_t NUM_PERIPHERALS = 2;
+
+  const Peripheral_t PERIPHERAL[NUM_PERIPHERALS] = {
+    { 50, A8 }, // Peripheral 0 
+    { 48, A9 }  // Peripheral 1
+  };
+
   const size_t NUM_MOTORS = 2;
```

Comment:
No, motors has nothing to do with peripherals. At this point motors really just represent the servo outputs whereas the peripherals represent the power outputs we use to power the pump and wifi currently.